### PR TITLE
Updated Readme

### PR DESCRIPTION
**Pull request by anurag8546**
Added new features to Readme.md

### CODE REVIEW

**Review by johnwalicki — CHANGES_REQUESTED** [review]
Thanks for the contribution.  This looks great.  Can you make 2 changes before we merge this.

- Remove the unnecessary `<br>` lines from your commit.
- Remove one of the trailing newlines from your commit.

Thanks so much for joining the Open Horizon community.

**Review by johnwalicki — COMMENTED** [review]
@joewxboy   Are we moving the CODE_OF_CONDUCT.md or CONTRIBUTING.md for all of the repositories into this .github profile?

**Comment by johnwalicki** [comment]
Some of the commits are messy multiple edits to links.  Maybe we could recommend to squash this into one commit ?

**Comment by johnwalicki** [comment]
@anurag8546  It would be great to merge your contribution into the Open Horizon project. See above for some requested changes. Can you git squash these commits and sign the DCO?

**Comment by anurag8546** [comment]
@joewxboy  I've done both the required changes -
1) Removed unnecessary <br> from code
2) Removed  the trailing newline from the commit.

This still gives all checks have failed.

**Comment by anurag8546** [comment]
> @joewxboy I've done both the required changes -
> 
> 1. Removed unnecessary  from code
> 2. Removed  the trailing newline from the commit.
> 
> This still gives all checks have failed.

@johnwalicki

File: profile/README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@
 <a href="https://github.com/open-horizon?tab=repositories"><img src="https://badges.pufler.dev/repos/open-horizon" alt="Open Horizon" /> </a>
 <img src="https://img.shields.io/github/stars/open-horizon?label=Stars" alt="Total Stars">
 <!--
-<a href="https://twitter.com/lf_edge)" alt="twitter"><img src="https://img.shields.io/badge/twitter-@lf_edge--twitter-yellow.svg" alt="twitter"></img></a>
+<a href="https://twitter.com/lf_edge" alt="twitter"><img src="https://img.shields.io/badge/twitter-@lf_edge--twitter-yellow.svg" alt="twitter"></img></a>
```
Comment (joewxboy) [inline profile/README.md]: Please remove closing `</img>` tag.  Image is not an HTML container, so it does not have a closing tag in HTML 4.x and HTML 5.

**Review by joewxboy — CHANGES_REQUESTED** [review]

**Comment by joewxboy** [comment]
@anurag8546 When you click on each individual commit, you are shown details about which checks are failing for that specific commit, and potentially how to resolve them.  Here's one:
https://github.com/open-horizon/.github/pull/17/checks

Let me know if the description on how to sign the commits is unclear or hard to follow.

**Comment by joewxboy** [comment]
@anurag8546 Are you available to amend your commits to add a sign-off?  That will allow this PR to be merged.  Thanks.

**Review by joewxboy — APPROVED** [review]

**Comment by joewxboy** [comment]
No DCO, closing
